index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Redressed&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+
+    <h1 id="title">freeCodeCamp Survey Form </h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p><br> 
+
+    <form id="survey-form">
+        <div>
+        <label id="name-label" for="name">Name</label><br>
+        <input type="text" id="name" placeholder='Enter Your Name'><br>
+        </div>
+
+        <div>
+        <label  id="email-label" for="email">Email</label><br>
+        <input type="email" id="email" name="email" pattern="x@domain.com" placeholder="Enter your Email"><br>
+        </div>
```

Comment:
- [ ] It is better to give more descriptive suggestions that some places in their code need to be changed.

Example: 
> Following the rule of sematic HTML makes your code much cleaner. It would be nice if you use semantic HTML instead of using the `div` tag. For more information:  [HTML Semantic Elements
](https://www.w3schools.com/html/html5_semantic_elements.asp)